conda_smithy/templates/travis.yml.tmpl

```diff
@@ -1,76 +1,117 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
 # update the conda-forge.yml and/or the recipe/meta.yaml.
 # -*- mode: yaml -*-
-
+{%- set platformset = [] %}
+{%- for _, platform, _ in configs %}
+  {%- set pfarchless = platform.split('-')[0] %}
+  {%- if pfarchless not in platformset %}
+    {%- do platformset.append(pfarchless) %}
+  {%- endif %}
+{%- endfor %}
 language: generic
 
-os: osx
-osx_image: {{ travis.get('osx_image', 'xcode9.4') }}
-
-
-{% block env -%}
-{% if configs[0] or travis.secure -%}
+{% if travis.secure -%}
 env:
-  {%- if configs[0] %}
-  matrix:
-    {%- for config, _, _ in configs %}
-    - CONFIG={{ config }}
-    {%- endfor %}
-  {%- endif %}
-{##}
-  {%- if travis.secure %}
   global:
     {%- for name, hashed_secure in travis.secure | dictsort %}
     # The {{ name }} secure variable. This is defined canonically in conda-forge.yml.
     - secure: "{{ hashed_secure }}"
     {%- endfor %}
-  {%- endif %}
 {%- endif %}
-{% endblock %}
+
+{% if configs[0] -%}
+matrix:
+  include:
+    {%- for config, platform, upload in configs | sort %}
+    - env: CONFIG={{ config }} UPLOAD_PACKAGES={{ upload }}
+    {%- if platform.startswith("osx") %}
+      os: osx
+      osx_image: xcode9.4
+    {%- elif platform.startswith("linux-ppc64le") %}
+      os: linux-ppc64le
```

Comment:
Add 
```
    {%- elif platform == "linux") %}
      os: linux
```
to make this complete.

Reply:
Are we wanting to start using Travis CI for generic Linux builds?

Reply:
If people wanted to, they could. It'll be easier for non conda-forge builds to set up only travis-ci

Reply:
conda-forge itself is very unlikely to, but the additional utility is nice for other build-out projects

Reply:
also done.